### ISSUE TITLE
[Question] how do i add two moons to one planet?

### ISSUE THREAD

**Issue by PercyJames** [zmaster587/AdvancedRocketry#1835]
## Version of Advanced Rocketry
1.12.2-1.7.0-221-universal
## Have you verified this is an issue in the latest unstable build
N/A
## Version of LibVulpes
1.12.2-0.4.2-75-universal
## Version of Minecraft
1.12.2
## Crash report or log (if applicable)
http://pastebin.com is a good place to put them
crash reports that are put in the issue itself are
hard to read
[insert here]

## Description of the problem
basically, i tried to add mars (Galacticraft) and succeeded however when i tried to add it's moons (Extraplanets) what happend when i put both moons in was weird. so, mars has phobos as a moon but phobos has demos as a moon.

https://pastebin.com/K2hrb6JA

it hath the issue

**Comment by MrEyeballs29** [comment]
First of all you forgot to close the tag. This is not a bug or a question you made a user error. here it is the correct syntax for both moons: https://pastebin.com/4w23YUC9

**Comment by PercyJames** [comment]
it worked!
